ignite/services/plugin/template/integration/integration_test.go.plush

```diff
@@ -0,0 +1,71 @@
+package integration_test
+
+import (
+	"bytes"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
Can we not mix match assert and require?
We can do assert.NoError as well or just use require for all of them